package.json

```diff
@@ -29,7 +29,7 @@
     "jsdom": "^21.1.1",
     "local-ssl-proxy": "^2.0.5",
     "pre-commit": "^1.2.2",
-    "prettier": "2.3.1",
+    "prettier": "^2.8.8",
```

Comment:
Why did you update the package json?

Reply:
Did it by mistake....resolved all the comments. Please review it